compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -2703,8 +2704,11 @@ impl<'a> Parser<'a> {
     }
 
     /// Parses the condition of a `if` or `while` expression.
+    ///
+    /// The specified `edition` should be that of the whole `if` or `while` construct: the same
+    /// span that we later decide the drop behaviour on (editions ..=2021 vs 2024..)
```

Comment:
```suggestion
    /// The specified `edition` should be that of the whole `if` or `while`
    /// construct, i.e. the same span we use to later decide whether the drop
    /// behavior should be that of `..=2021` or that of `2024..`, as this is
    /// important for let chains.
```

(Mostly wanted to add the trailing period, but then reworded a bit for hopefully better clarity.)

Reply:
reworded